changelog.md

```diff
@@ -1,3 +1,36 @@
+# 3.8.0 / 2021-04-27
+
+### New Features
+
+- Content can be added to personnal favourites
+- Add and comment a publication
+- Home/login page can be customizable 
+- Preview of links can be genearated in comment
+- Be able to fold the subspaces in the sidebar
+- New space dashboard
+- Possibilities to modify comments
+- New navigation links to user profile
+- Add button "open the gallery" on the content page
+- Allow to open file uploaded in a comment as a content
+- Rich text available in dashboard description
+
+### Fixed Issues
+[#](https://github.com/tracim/tracim/issues/)
```

Comment:
is this "generic" line normal?

Reply:
no :(

Reply:
Done

Reply:
ok, so I little suggestion to apply :smile: 
```suggestion
```